pkgs/development/libraries/pango/default.nix

```diff
@@ -31,6 +31,8 @@ stdenv.mkDerivation rec {
 
   configureFlags = optional stdenv.isDarwin "--without-x";
 
+  doCheck = false; # fails 1 out of 12 tests with "Fontconfig error: Cannot load default config file"
```

Comment:
I wonder if `makeFontsConf` would help with this… (still a mergeable improvement as-is, of course)